src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,188 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> node;
```

Comment:
Please try to rework solution without this variable.
Variable isolated within methods are much more reliable in work
```suggestion
```

Reply:
Done. One of the reasons I did it before is to escape code duplication. Thx for feedback.